From: Dave Jones <davej@redhat.com>
Subject: x86: Remove the CPU cache size printk's
References: bnc#564618, FATE#306952
Patch-mainline: 2.6.33-rc1
Git-commit: 15cd8812ab2ce62a2f779e93a8398bdad752291a

They aren't really useful, and they pollute the dmesg output a lot
(especially on machines with many cores).

Also the same information can be trivially found out from
userspace.

Reported-by: Mike Travis <travis@sgi.com>
Signed-off-by: Dave Jones <davej@redhat.com>
Acked-by: H. Peter Anvin <hpa@zytor.com>
LKML-Reference: <20091112231542.GA7129@redhat.com>
Signed-off-by: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Rafael J. Wysocki <rjw@suse.de>
---
 arch/x86/kernel/cpu/intel_cacheinfo.c |   16 ----------------
 1 file changed, 16 deletions(-)

Index: linux-2.6.32-master/arch/x86/kernel/cpu/intel_cacheinfo.c
===================================================================
--- linux-2.6.32-master.orig/arch/x86/kernel/cpu/intel_cacheinfo.c
+++ linux-2.6.32-master/arch/x86/kernel/cpu/intel_cacheinfo.c
@@ -491,22 +491,6 @@ unsigned int __cpuinit init_intel_cachei
 #endif
 	}
 
-	if (trace)
-		printk(KERN_INFO "CPU: Trace cache: %dK uops", trace);
-	else if (l1i)
-		printk(KERN_INFO "CPU: L1 I cache: %dK", l1i);
-
-	if (l1d)
-		printk(KERN_CONT ", L1 D cache: %dK\n", l1d);
-	else
-		printk(KERN_CONT "\n");
-
-	if (l2)
-		printk(KERN_INFO "CPU: L2 cache: %dK\n", l2);
-
-	if (l3)
-		printk(KERN_INFO "CPU: L3 cache: %dK\n", l3);
-
 	c->x86_cache_size = l3 ? l3 : (l2 ? l2 : (l1i+l1d));
 
 	return l2;
